wicket-native-websocket/wicket-native-websocket-core/src/main/java/org/apache/wicket/protocol/ws/api/IWebSocketConnection.java

```diff
@@ -55,6 +56,27 @@
 	 */
 	IWebSocketConnection sendMessage(String message) throws IOException;
 
+    /**
+     * Sends a text message to the client.
```

Comment:
Please add an explanation why/how this method is different than `#sendMessage()`

Reply:
Done